app/client/views/shared/VueComponent/VueComponent.js

```diff
@@ -0,0 +1,5 @@
+Template.BlazeVueComponent.helpers({
+  args() {
+    return { filters: { localize: TAPi18n.__ } };
```

Comment:
Can this be done as a regular [Vue mixin](https://vuejs.org/v2/guide/mixins.html)? I.e. it is just an object that is imported and assigned to the Vue instance `mixins` array, like:

```js
// in localizationMixin.js
export default { filters: { localize: TAPi18n.__ } };
```

```js
import localizationMixin from 'localizationMixin';

export default {
  mixins: [localizationMixin],
  props: {
    Modal: Object
  }
};
```

Would something close to that work? It just seems a bit mysterious to me how the VueComponent mixin approach is working, and I would like to stick as close as possible to the Vue docs.

Reply:
The localization mixin might go in a mixins folder, like so: `app/client/mixins`